### DISCUSSION TITLE
How do I use main.css to reuse styles?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [nativewind/nativewind#447] [Q&A]
How can I reuse styles, I want to have this kind of main.css file that I can later use in my components. Is this possible with nativewind? 

```
      <TextInput
        className="basic-input"
        placeholder="Email"
        autoCapitalize="none"
        onChangeText={setEmail}
      />
```

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer components {
  .basic-input {
        @apply w-full h-10 px-2 border border-gray-300 rounded-md;
  }
}
```

**Comment by zerodeleo** [comment]
Looks like a solution for this will be released in v4, [docs](https://www.nativewind.dev/v4/overview)

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
